requirements.txt

```diff
@@ -0,0 +1,13 @@
+attrs
+databroker
+doct
+doctr
+jinja2
+lmfit
+mongoquery
+networkx
+nose
+ophyd
+pims
+pyepics
+zmq
```

Comment:
I think you want [pyzmq](https://pypi.python.org/pypi/pyzmq) also is this a requirement for xpdAcq?